pages/components/docs/Dropdown.md

```diff
@@ -3,26 +3,44 @@
 
 The Dropdown component is a lightweight context menu for housing navigation and actions.
 
+Dropdown.Button is used to trigger opening and closing the dropdown.
+
+Dropdown.Menu wraps your menu content. Be sure to pass a `direction` prop to this component to position the menu in relation to the Dropdown.Button.
+
 ## Default example
 ```.jsx
 <Dropdown scheme="primary" minWidth="5em">
-  <Box role="list">
-    <Box role="listitem">Item 1</Box>
-    <Box role="listitem">Item 2</Box>
-    <Box role="listitem">Item 3</Box>
-  </Box>
+  <Dropdown.Button>Dropdown </Dropdown.Button>
+  <Dropdown.Menu direction='sw'>
+    <Dropdown.Item>Item 1</Dropdown.Item>
+    <Dropdown.Item>Item 2</Dropdown.Item>
+    <Dropdown.Item>Item 3</Dropdown.Item>
+  </Dropdown.Menu>
 </Dropdown>
 ```
 
 ## System props
 
-Dropdown components get `COMMON` system props. Read our [System Props](/components/docs/system-props) doc page for a full list of available props.
+Dropdown wrapper components get `COMMON` system props. Read our [System Props](/components/docs/system-props) doc page for a full list of available props.
+
+Dropdown.Menu, Dropdown.Button, and Dropdown.Item do not get any system props.
 
 ## Component props
 
+
+#### Dropdown.Menu
 | Name | Type | Default | Description |
 | :- | :- | :-: | :- |
-| scheme | String | | Button scheme used to style the component, can be one of `danger`, `outline` or `primary` |
-| title | String | | Optional prop to set the text in the Dropdown button
+| direction | String | 'sw' | Sets the direction of the dropdown menu. |
```

Comment:
Can't wait until this documentation is automated! 😄

Reply:
Same 😭